deployment/src/main/java/io/quarkiverse/quinoa/deployment/packagemanager/types/PackageManagerType.java

```diff
@@ -10,7 +10,7 @@ public enum PackageManagerType {
     // Order matters for detection
     PNPM("pnpm", "pnpm-lock.yaml", "install --frozen-lockfile"),
     NPM("npm", "package-lock.json", "ci"),
-    YARN("yarn", "yarn.lock", "install --frozen-lockfile"),
+    YARN("yarn", "yarn.lock", "install --immutable"),
```

Comment:
By doing this you would break yarn 1

Reply:
Agreed. I just can't make any of these tests pass on windows.  I fix one thing then another yarn test fails

Reply:
I might have a solution, trying something